tests/keras_tests/function_tests/test_export_keras_fully_quantized_model.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2022 Sony Semiconductor Israel, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+import os
+import random
+import unittest
+
+import keras.models
+import numpy as np
+import tensorflow as tf
+from tensorflow.keras.applications.mobilenet_v2 import MobileNetV2
+
+import model_compression_toolkit as mct
+from model_compression_toolkit.exporter.target_platform_export.keras import export_keras_fully_quantized_model, \
+    KerasExportMode
+from tests.keras_tests.tpc_keras import get_activation_quantization_disabled_keras_tpc
+
+tp = mct.target_platform
+
+SAVED_MODEL_PATH = '/tmp/exported_tf_fakelyquant.h5'
+
+
+class TestExporter(unittest.TestCase):
+
+    def tearDown(self):
+        os.remove(SAVED_MODEL_PATH)
+
+    def setUp(self) -> None:
+        self.mbv2 = MobileNetV2()
+        self.representative_data_gen = lambda: [np.random.randn(1, 224, 224, 3)]
```

Comment:
This seems like a large test, how long does it run?
If it is too long maybe add a unit test (with a small model and input) so it would be simpler to run it in the test suit.

Additionally, you might want to test the exporter with other complicated layers that are not part of MobilenetV2? if so, several small unit tests would do the job I think better than one large MBV2 test.